test/Shape.test.ts

```diff
@@ -0,0 +1,44 @@
+import * as THREE from 'three';
+import Shape from '../src/geometry/Shape';
```

Comment:
Use `import { Geometry } from ../src/index.js' and then use `Geometry.Shape` to be consistent with the other files. Also, this file doesn't import the test functions from vitest which causes it to fail with `bun run test`.